worker/worker.py

```diff
@@ -336,6 +337,12 @@ def main():
         dest="use_all_cores",
         default=config.get("parameters", "use_all_cores"),
     )
+    parser.add_option(
+        "-w",
+        "--write_config",
+        dest="write_config",
+        default=config.get("parameters", "write_config"),
```

Comment:
`# default=config.get("parameters", "write_config"),`

Reply:
Tonight I will have more time to try/test.

Reply:
@tomtor `optparse` is deprecated, suggested `argparse` (but this could break python2), see https://docs.python.org/3/library/optparse.html

This example use `argparse` and `configparse`
http://www.jtrive.com/developing-command-line-applications-with-argparse-and-configparser.html

Reply:
Fixed

Reply:
@ppigazzini We still have some Python 2 workers....

Reply:
@tomtor you sometimes uses a python 2 worker :)

Reply:
Yes, by accident...

Reply:
Python 2 has `argparse` but we should learn the hard way the usage differences wrt Python 3: too much pain :)
https://docs.python.org/2/howto/argparse.html